webui/react/src/stores/cluster.tsx

```diff
@@ -60,7 +60,7 @@ export const maxPoolSlotCapacity = (pool: ResourcePool): number => {
    * zeroes, and this would at least unblock experiments, and the backend would be able
    * to return capacity issues.
    */
-  return pool.slotsAvailable || Infinity;
+  return pool.slotsAvailable;
```

Comment:
This seems like a good change! Do we also need to make changes elsewhere to make sure we don't do `0/0` and get `NaN` or other unexpected results? Looks like at least [here](https://github.com/determined-ai/determined/blob/main/webui/react/src/components/SlotAllocationBar.tsx#L130), we divide by the value returned by this function.

Reply:
Now that I think about it, a more conservative way of changing this could be to have 'maxClusterSlotCapacity' ignore when 'maxPoolSlotCapacity' returns 'Infinity'.

Still leaves the door open to other potential issues due to 'Infinity', but also more definitively avoids 'NaN' due to dividing by zero 🤷

Reply:
Ok actually if there are no slots, I don't think we should artificially change that to Infinity just to avoid a specific error. Instead, anytime we divide, we should handle the possibility of the denominator being 0.

Reply:
Ok actually if there are no slots, I don't think we should artificially change that to Infinity just to avoid a specific error. Instead, anytime we divide, we should handle the possibility of the denominator being 0, which I believe we're doing for usage of 'maxPoolSlotCapacity' anyway.